### PR TITLE
fix/c2 flow fixes

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

**Review by nklomp — APPROVED** [review]

File: src/services/pexService.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ export const getMatchingPidCredentials = async ({
   const credentials = pidCredentials.map(pidCredential => {
     const dc: NonPersistedDigitalCredential = nonPersistedDigitalCredentialEntityFromAddArgs({
       rawDocument: pidCredential.rawCredential,
-      kmsKeyRef: 'fixme', // FIXME
+      kmsKeyRef: 'https://demo.pid-issuer.bundesdruckerei.de/c2/.well-known/openid-credential-issuer',
```
Comment (nklomp) [inline src/services/pexService.ts]: How does this work when using the regular C flow?

**Review by nklomp — CHANGES_REQUESTED** [review]
Not sure the C flow still works mainly when sharing with a RP. If you ask me the static kmsKeyRef will cause issues

File: src/services/pexService.ts (hunk 1 above)
Comment (BtencateSphereon) [reply src/services/pexService.ts]: not ignored, but i am waiting on a new ssi-sdk version to test everything

Comment (BtencateSphereon) [reply src/services/pexService.ts]: Seems to be working for in all flows. but i made it a condition to only set the kmsKeyRef for the c2 flow. would that be ok for now?

**Pull request merged**